File: VeryGoodViroGame/src/VeryGoodViroGame/Field/FieldLabBear.java
package VeryGoodViroGame.Field;

import VeryGoodViroGame.Agent.Bear;
import VeryGoodViroGame.GameManager;
import VeryGoodViroGame.Logger;
import VeryGoodViroGame.Virologist;

public class FieldLabBear extends FieldLab
{
    public FieldLabBear()
    {
        Map curmap = GameManager.GetMap();
        if(curmap != null)
        {
            curmap.Swallow(this, false);
        }
    }
    @Override
    public void Interact(Virologist v)
    {
        Bear b = new Bear();
        b.Apply(null, v);
    }
    @Override
    public String GetDrawString()
    {
        return "bearlab";
    }
}

File: tervezes/uml_gen/uml_generator/FieldLabBear.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace VeryGoodViroGame
{
    public class FieldLabBear : FieldLab
    {
        public override void Interact(Virologist v)
        {
            Logger.NewFunctionCall(this, "Interact");
            Bear b = new Bear();
            b.Apply(null, v);
            Logger.ReturnFunction();
        }
    }
}